ScoreSaber/UI/Leaderboard/ScoreSaberLeaderboardViewController.cs

```diff
@@ -81,98 +68,96 @@ public enum UploadStatus {
-                Application.OpenURL($"https://scoresaber.com/leaderboard/{_leaderboardService.currentLoadedLeaderboard.leaderboardInfoMap.leaderboardInfo.id}");
-            };
+        [UIObject("loadingLB")]
+        private readonly GameObject loadingLB;
 
-            _panelView.rankingWasSelected = delegate () {
-                _parserParams.EmitEvent("close-modals");
-                _parserParams.EmitEvent("show-profile");
-                _profileDetailView.ShowProfile(_playerService.localPlayerInfo.playerId).RunTask();
-            };
+        [UIObject("starRatingBox")]
+        private readonly GameObject starRatingBox;
 
-            _container.Inject(_profileDetailView);
-            _playerService.GetLocalPlayerInfo();
-        }
+        [UIValue("map-info-view")]
+        protected MapInfoView _mapInfoView = null;
 
-        private void LeaderboardViewDeactivate(bool removedFromHierarchy, bool screenSystemDisabling) {
-            if (_scoreDetailView.detailModalRoot != null) _scoreDetailView.detailModalRoot.Hide(false);
-            if (_profileDetailView.profileModalRoot != null) _profileDetailView.profileModalRoot.Hide(false);
-        }
+        [UIAction("OnPageUp")] private void UpButtonClicked() => UpdatePageChanged(-1);
+        [UIAction("OnPageDown")] private void DownButtonClicked() => UpdatePageChanged(1);
+
+
+        public bool activated { get; private set; }
+        public int leaderboardPage { get; set; } = 1;
+
+        public ScoreSaberScoresScope currentScoreScope { get; set; }
+
+        private bool _replayDownloading = false;
+        private string _currentLeaderboardRefreshId = string.Empty;
+        private BeatmapKey _currentBeatmapKey;
+
+        Color yellow = new Color(250f / 255f, 221f / 255f, 45f / 255f);
+        Color green = new Color(63 / 255f, 191 / 255f, 100 / 255f);
+        Color grey = new Color(125f / 255f, 125f / 255f, 125f / 255f);
+        Color blue = new Color(62 / 255f, 152 / 255f, 237 / 255f);
+        Color pink = new Color(235 / 255f, 73 / 255f, 232 / 255f);
+        Color _scoreSaberBlue = new Color(0f, 0.4705882f, 0.7254902f);
+
+
+        [Inject] private readonly PanelView _panelView;
+        [Inject] private readonly SiraLog _log;
+        [Inject] private readonly DiContainer _container;
+        [Inject] private readonly IUploadDaemon _uploadDaemon;
+        [Inject] private readonly ReplayLoader _replayLoader;
+        [Inject] private readonly PlayerService _playerService;
+        [Inject] private readonly LeaderboardService _leaderboardService;
+        [Inject] private readonly PlayerDataModel _playerDataModel;
+        [Inject] internal readonly PlatformLeaderboardViewController _platformLeaderboardViewController;
+        [Inject] internal readonly ScoreSaberRichPresenceService _scoresaberRichPresence;
+        [Inject] private readonly MaxScoreCache _maxScoreCache;
+        [Inject] private readonly PlatformLeaderboardViewController _plvc;
```

Comment:
variable name does not fit the other names here. Should be _platformLeaderboardViewController